Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1172,7 +1172,10 @@ def logger(func):
 
     def func_wrapper(*args, **kwargs):
         LOG('calling {}({})'.format(func.__name__, formatAllArgs(args, kwargs)))
-        return func(*args, **kwargs)
+        ret_val = func(*args, **kwargs)
+        if is_debug_mode():
+            demisto.debug('Return value: {}'.format(str(ret_val)))
```

Comment:
Shouldn't we use `LOG` here? Maybe also if in debug_mode add an immediate flush.

Reply:
@glicht doesnt the flush occur in the `DebugLogger` anyway?